templates/questionnaire.mustache

```diff
@@ -84,13 +85,14 @@
     {{#ratedquestions}}
         <tr>
             <td>
-                <div class="row-fluid rtl-compatible" data-region="question-row" data-itemid="{{id}}">
+<!--                 <div class="row-fluid rtl-compatible" data-region="question-row" data-itemid="{{id}}"> -->
+                <div class="row rtl-compatible" data-region="question-row" data-itemid="{{id}}">
                     <div class="col-md-5 span5">
                         {{question}}
                     </div>
                     {{#scales}}
                         <div class="col-md-1 span1 scaleoption text-center">
-                            <label class="scaleoptionlabel label label-default p-l-1 p-r-1" data-value="{{scale}}" tabindex="0" for="option-{{questionid}}-{{scale}}" title="{{description}}">
+                            <label class="scaleoptionlabel badge badge-secondary text-light p-l-1 p-r-1" data-value="{{scale}}" tabindex="0" for="option-{{questionid}}-{{scale}}" title="{{description}}">
```

Comment:
Fix to:
`<label class="scaleoptionlabel badge bg-secondary text-light p-l-1 p-r-1" data-value="{{scale}}" tabindex="0" for="option-{{questionid}}-{{scale}}" title="{{description}}">`